GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -345,7 +347,12 @@ - (void)pageEventDetected:(NSNotification *)notification {
         
         // dismiss a follow up modal if it is present and displayed
         if (self.followupViewController) {
-            [self dismissViewControllerAnimated:YES completion:nil];
+            self.bypassPresentingNavbar = YES;
+            __weak typeof(self) weakSelf = self;
+            
+            [self dismissViewControllerAnimated:YES completion:^{
```

Comment:
I would call this method on self.followupViewController instead of self. dismissViewControllerAnimated either will dismiss the view controller the current view controller presented or if there is no presented view controller it will ask the parent view controller to dismiss the current view controller. calling the method on self.followupViewController means there is no chance of you accidentally dismissing the current view controller if the state got messed up. https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIViewController_Class/#//apple_ref/occ/instm/UIViewController/dismissViewControllerAnimated:completion: